stress-testing/stress-test.py

```diff
@@ -2,68 +2,77 @@
 import argparse
 import uuid
 import time
+from contextlib import asynccontextmanager
 
 from jupyterhub_client.api import JupyterHubAPI, JupyterAPI
 from jupyterhub_client.utils import parse_notebook_cells, tangle_cells
 
+
+@asynccontextmanager
+async def use_user_server(hub, username):
+    """
+    Start a server for username on hub, cleaning up when done
+    """
+    try:
+        await hub.ensure_user(username, create_user=True)
+        print(f'{username}: created user')
+        try:
+            await hub.ensure_server(username, create_user=False)
```

Comment:
Great idea to call both `hub.ensure_user` and `hub.ensure_server`, since if the user exists, `hub.ensure_server` won't do anything about it, just an extra call for confirmation <3

Reply:
Yep, exactly! Plus, splitting this up lets us split 'delete_server' from 'delete_user', to be sure that delete user runs regardless of issues with stopping server.